src/Schema/Resolver/ResolverRegistry.php

```diff
@@ -44,7 +44,7 @@ public function getFieldResolver(string $typeName, string $fieldName): ?callable
         $resolver = $this->getResolver($typeName);
 
         $resolver = $resolver instanceof ResolverCollection
-            ? $resolver->getResolveCallback()($fieldName)
+            ? $resolver->getResolver($fieldName)
```

Comment:
This was actually a bug that I found while working on this.

Reply:
👍